elasticsearch-model/examples/activerecord_mapping_rendering_paperclip_attachment_urls.rb

```diff
@@ -0,0 +1,158 @@
+require 'ansi'
+require 'sqlite3'
+require 'active_record'
+require 'elasticsearch/model'
+require 'paperclip'
+
+require "paperclip/railtie"
+Paperclip::Railtie.insert
+
+ActiveRecord::Base.logger = ActiveSupport::Logger.new(STDOUT)
+ActiveRecord::Base.establish_connection( adapter: 'sqlite3', database: ":memory:" )
+
+ActiveRecord::Schema.define(version: 1) do
+  create_table :properties do |t|
+    t.string :title
+    t.string :description
+    t.timestamps
+  end
+
+  create_table :images do |t|
+    t.integer :property_id
+    t.string :attachment_file_name
+    t.timestamps
+  end
+end
+
+begin
+  class Image < ActiveRecord::Base
+    belongs_to :property, touch: true
+    has_attached_file :attachment, styles: { thumb: '100x100>', screen: '1024x1024' }
+  end
+rescue => e
+  puts e.inspect.ansi(:bold, :red)
```

Comment:
Paparclip is nasty, in some cases it may not be loaded properly and the `has_attached_file` will report `NoMethodError`